config/jest/jest.config.js

```diff
@@ -1,10 +1,9 @@
-
 module.exports = {
-    'rootDir': '../../',
+    'rootDir': '../../src/',
```

Comment:
limit to `src` directory so that visual regression tests are not run on every commit/every test run